service/indexer.go

```diff
@@ -0,0 +1,24 @@
+package service
+
+import (
+	"path/filepath"
+
+	"go.vocdoni.io/dvote/log"
+	"go.vocdoni.io/dvote/vochain/indexer"
+)
+
+// VochainIndexer creates the vochain indexer service.
+func (vs *VocdoniService) VochainIndexer() error {
+	log.Info("creating vochain indexer service")
+	var err error
+	if vs.Indexer, err = indexer.NewIndexer(
+		filepath.Join(vs.Config.DataDir, "indexer"),
+		vs.App,
+		!vs.Config.Indexer.IgnoreLiveResults,
+	); err != nil {
+		return err
+	}
+	// launch the indexer after sync routine (executed when the blockchain is ready)
+	go vs.Indexer.AfterSyncBootstrap()
+	return nil
+}
```

Comment:
given the multiline, i feel it would be more readable (idiomatic even?) to just do 
```
vs.Indexer, err := ...(multiline)...
if err != nil {
  return err
}
```
and also you don't need the explicit `var err error`

Reply:
Thanks for the observation, but I'm not certain why it would be more idiomatic.

We usually try to use inline `if` when possible. And we also try to declare the vars in the most reduced scope.

These are the two rules that we are usually following. They are properly applied here.